packages/pigeon/lib/objc_generator.dart

```diff
@@ -232,47 +234,67 @@ NSObject<FlutterMessageCodec>* ${_getCodecGetterName(options.prefix, api.name)}(
 ''');
 }
 
+String _spaceJoin(String x, String y) => '$x $y';
+
+String _makeObjcSignature({
+  required Method func,
+  required ObjcOptions options,
+  required String returnType,
+  required String lastArgLabel,
+  required String lastArg,
+  Iterable<String>? argNames,
```

Comment:
This is the same construction that we agreed not to use in the Dart code; it should be changed here as well.

Reply:
Done.